packages/consumption/test/modules/requests/IncomingRequestsController.test.ts

```diff
@@ -412,9 +413,10 @@ describe("IncomingRequestsController", function () {
             await Given.aTerminatedRelationshipToIdentity();
             await Given.anIncomingRequestInStatus(LocalRequestStatus.DecisionRequired);
             const validationResult = await When.iCallCanAccept();
-            expect(validationResult).errorValidationResult({
-                code: "error.consumption.requests.wrongRelationshipStatus"
-            });
+            expect((validationResult as ErrorValidationResult).error.code).toBe("error.consumption.requests.wrongRelationshipStatus");
+            expect((validationResult as ErrorValidationResult).error.message).toContain(
+                "You cannot decide a request from 'did:e:a-domain:dids:anidentity' since the relationship is in status 'Terminated'."
+            );
```

Comment:
you made this code harder to read. please go back to errorValidationResult as this allows checking code and message in one call **and** additionally checks that

edit, I've done it and pushed it.